### DISCUSSION TITLE
Actions runner image built with custom runner code not working as expected

### DISCUSSION THREAD

**Discussion by rpall08** [actions/actions-runner-controller#1858] [Questions]
Trying to create actions runner image with custom runner code, below are changes in **actions-runner.dockerfile**. 
copies custom built runner tag.gz file in **actions-runner-controller/runner** directory.

COPY ./runner.tar.gz /runnertmp/runner.tar.gz
ENV RUNNER_ASSETS_DIR=/runnertmp
RUN mkdir -p "$RUNNER_ASSETS_DIR" \
    && cd "$RUNNER_ASSETS_DIR" \
    && tar xzf ./runner.tar.gz \
    && rm runner.tar.gz \
    && ./bin/installdependencies.sh \
    && mv ./externals ./externalstmp \
    && apt-get install -y libyaml-dev \
    && rm -rf /var/lib/apt/lists/*

Image built fine and then tried to start the container with these environment properties
RUNNER_TOKEN=******
RUNNER_ORG=******
RUNNER_NAME=**********

**logs from runner container:**

DEBUG --- Working with public GitHub
ERROR --- /runner should be an emptyDir mount. Please fix the pod spec.

Can anyone review, whats wrong in dockefile?

**Comment by rpall08** [comment]
@mumoshu , could you suggest whats wrong with custom runner docket file?

**Comment by bm1216** [comment]
Hey, 👋🏽 
What does your runner spec look like?
